seed.yaml.sample

```diff
@@ -0,0 +1,10 @@
+Keys:
+  Asymmetric:
+    Rsa:
+      - Metadata:
+          KeyId: "REPLACE_ME_KEY_ID_HERE"
+          KeyUsage: SIGN_VERIFY
+          Description: RSA key with 2048 bits
+        PrivateKeyPem: |
+          -----BEGIN PRIVATE KEY-----
```

Comment:
Urgh GitGuardian is emailing me to say I've _leaked a private key_ because of this commit.

Guess we know how their Regex works now.